lib/literature/components/forms/post_form_component.ex

```diff
@@ -51,8 +51,8 @@ defmodule Literature.PostFormComponent do
               phx-hook="EditorJS"
             >
             </div>
-            <%= hidden_input(f, :editor_json) %>
-            <%= hidden_input(f, :html) %>
+            <input type="hidden" name="post[editor_json]" />
+            <input type="hidden" name="post[html]" />
```

Comment:
I don't think we should hardcode the name attributes because it relies on the `as` of the form. Maybe we can do this instead:
```
<input type="hidden" name={f[:editor_json].name} />
<input type="hidden" name={f[:html].name} />
```

Not really sure if we need the to_form/1 to use it, but we can try as well:
```
<.form
        :let={f}
        for={to_form(@changeset)}
        id="post-form"
        multipart
        phx-target={@myself}
        phx-change="validate"
        phx-submit="save"
      >
...
```